_includes/configuration/iframe-aria-text.md

```diff
@@ -0,0 +1,24 @@
+## `iframe_aria_text`
+
+{{ site.requires_5_9v }}
+
+This option is used to customize the `title` attribute on the {{site.productname}} `iframe` element. For example:
+ 
+```html
+<iframe title="Rich Text Area. Press ALT-0 for help."></iframe>
+```
+
+The `title` attribute is read by screen-readers to help users identify the editor. This option only applies to {{site.productname}} classic (iframe) mode and has no effect on [`inline` editors]({{site.baseurl}}/configure/editor-appearance/#inline).
```

Comment:
Should we mention, for inline mode, the developer has free reign to set the aria title on the content

Reply:
TMI?

Reply:
I'm going to put that in the 'out of scope' bucket.